### DISCUSSION TITLE
Nginx Proxy manager failed to update

### DISCUSSION THREAD

**Discussion by JaviSolo** [tteck/Proxmox#3069] [Help Needed]
Hi all, 

Im trying to use bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/nginxproxymanager.sh)" to update my LXC and after use it it's prompting me this error
![imagen](https://github.com/tteck/Proxmox/assets/137202353/3eea72f7-3d8c-4b64-8c80-414bc7b4e517)

Any idea about what's happening and how to solve it? As you can see, Im also trying the "update" command as the guide says and looks like the container doesn't recognize the command.

Thanks in advance

**Comment by tteck** [comment]
Maybe this will help, https://github.com/tteck/Proxmox/issues/2706#issuecomment-2041056086
